### PR TITLE
Refactor and include datasource manifest test

### PR DESCRIPTION
**Pull request by shreyabiradar07**
## Description

This PR has the following changes 

-  Supports new  input parameters - datasource `servicename` and `namespace` to test the datasource manifest configuration.
- Use `resource_optimization_local_monitoring.json` from autotune manifests and delete the JSON files from the test folder
- Update MetricProfile name referred in createExp test JSONs
- Remove empty `version` field from 'resource_optimization_local_monitoring.json`

Test results on ResourceHub Cluster:
```
./tests/test_autotune.sh -c openshift -i quay.io/shbirada/port-configuration:openshift  --testsuite=local_monitoring_tests --servicename=prometheus-k8s --datasource_namespace=openshift-monitoring 
```
[PR1291_openshift_local_monitoring_tests.zip](https://github.com/user-attachments/files/17010098/PR1291_openshift_local_monitoring_tests.zip)

### Type of change

- [ ] Bug fix
- [x] New feature
- [ ] Docs update
- [ ] Breaking change (What changes might users need to make in their application due to this PR?)
- [ ] Requires DB changes

## How has this been tested?

Please describe the tests that were run to verify your changes and steps to reproduce. Please specify any test configuration required. 

- [ ] New Test X
- [ ] Functional testsuite

**Test Configuration**
* Kubernetes clusters tested on: 

## Checklist :dart:

- [x] Followed coding guidelines
- [x] Comments added
- [ ] Dependent changes merged
- [ ] Documentation updated
- [x] Tests added or updated

## Additional information

.

### CODE REVIEW

File: tests/scripts/local_monitoring_tests/rest_apis/test_create_metric_profile.py [hunk 1]
```diff
@@ -25,6 +25,12 @@
 from helpers.utils import *
 from helpers.list_metric_profiles_validate import *
 from helpers.list_metric_profiles_without_parameters_schema import *
+from pathlib import Path
+
+current_directory = Path(__file__).resolve().parent
```
Comment (chandrams) [inline tests/scripts/local_monitoring_tests/rest_apis/test_create_metric_profile.py:30]: @shreyabiradar07 Please add a common function in utils to get the metric profile dir

Comment (shreyabiradar07) [reply tests/scripts/local_monitoring_tests/rest_apis/test_create_metric_profile.py]: Updated common function - `get_metric_profile_dir()` in `utils.py`

**Review by chandrams — APPROVED** [review]
LGTM

**Comment by chandrams** [comment]
LGTM

**Pull request merged**